pkg/platform/kube/getter.go

```diff
@@ -46,22 +46,27 @@ func (g *getter) get(consumer *consumer, getOptions *platform.GetOptions) ([]pla
 	functioncrInstances := []functioncr.Function{}
 
 	// if identifier specified, we need to get a single function
-	if getOptions.Name != "" {
+	if len(getOptions.MatchCriterias) > 0 {
 
-		// get specific function CR
-		function, err := consumer.functioncrClient.Get(getOptions.Namespace, getOptions.Name)
-		if err != nil {
+		// Iterate for every function name in MatchCriterias
+		for funcIndex := 0; funcIndex < len(getOptions.MatchCriterias); funcIndex++ {
+
+			// get specific function CR
+			function, err := consumer.functioncrClient.Get(getOptions.Namespace,
+				getOptions.MatchCriterias[funcIndex].Name)
+			if err != nil {
 
-			// if we didn't find the function, return an empty slice
-			if apierrors.IsNotFound(err) {
-				return functions, nil
+				// if we didn't find the function, return an empty slice
+				if apierrors.IsNotFound(err) {
```

Comment:
This looks like incorrect behavior - previously, if the `nuctl get` command included an identifier it meant users wanted to see one specific function. If it doesn't exist, it makes sense to return the empty slice. However now, whenever more than one identifier is given to `nuctl get`, we need to keep looking for any other function.

In short:
`nuctl get function` -> list all functions
`nuctl get function x, y` when x, y and z exist -> list x and y
`nuctl get function x, y` when only x exists -> list x